frontend-react/src/App.tsx

```diff
@@ -70,7 +70,6 @@ const App = () => {
                         <ReportStreamHeader />
                         <Switch>
                             <Route path="/" exact={true} component={Home} />
-                            <SecureRoute path="/daily" component={Daily} />
```

Comment:
This will remove /daily as a secure route from the ReactRouter - was this what was intended?

Reply:
@Bisonburger 
it is now an "AuthorizedRoute", protected with "SecureRoute" _**and**_ a permission. This way, if a user does not have receiving permissions, they cannot go to this route

Reply:
@Bisonburger 
https://github.com/CDCgov/prime-reportstream/blob/62071be985a37d1c48919fc4b91e01089290d004/frontend-react/src/App.tsx#L96
this is where it is now an AuthorizedRoute